src/main/java/io/github/fusionflux/portalcubed/content/PortalCubedSounds.java

```diff
@@ -25,6 +29,8 @@ public class PortalCubedSounds {
 	public static final SoundEvent PORTAL_GUN_HOLD_LOOP = register("portal_gun_hold_loop");
 	public static final SoundEvent PORTAL_GUN_DROP = register("portal_gun_drop");
 
+	public static final SoundEvent SURPRISE = register("pedestal_button_easter_egg");
```

Comment:
Make this a generic surprise sound

Reply:
Blame carter for making the id pedestal button specific :ioa: